packages/suite/src/reducers/suite/metadataReducer.ts

```diff
@@ -287,4 +292,19 @@ export const selectIsLabelingAvailableForEntity = (
     );
 };
 
+export const selectPasswordManagerState = (
+    state: {
+        metadata: MetadataState;
+    },
+    fileName?: string,
+) => {
+    const provider = selectSelectedProviderForPasswords(state);
+
+    if (!fileName || !provider || !provider?.data?.[fileName]) {
+        return;
+    }
+
+    return provider.data[fileName] as PasswordManagerState;
```

Comment:
the type of `provider.data[fileName]` should look like this [7d17cd7](https://github.com/trezor/trezor-suite/pull/10439/commits/7d17cd74007c49132077d321569360ce038a8272)

it would requrie some typeguard / type assertion here.